apps/dashboard/src/components/create-workflow-button.tsx

```diff
@@ -54,6 +55,12 @@ export const CreateWorkflowButton = (props: CreateWorkflowButtonProps) => {
       setIsOpen(false);
     },
   });
+  const { data: tagData } = useQuery<{ data: { name: string }[] }>({
```

Comment:
Let's factor this out to a dedicated `useTags` hook. We will need it in other screens as well.

Reply:
Done

Reply:
@desiprisg I'm totally fine with following `useXQuery, useXMutation` pattern ;)

Reply:
Folks, the one we have now is OK as well. So please align them separately.

Reply:
I also meant not passing `onSuccess` and `onError` funcs, and just passing through what `@tanstack/react-query` accepts.